web/src/features/moreCast2/components/MoreCast2Column.tsx

```diff
@@ -75,209 +75,50 @@ export class DateForecastField implements ColDefGenerator {
-          WindDirForecastField.precision,
-          new GridComponentRenderer()
-        )
-      )
-    }
-
-    return WindDirForecastField.instance
+    return this.colDefBuilder.generateColDefs(headerName ?? this.headerName, this.includeBias)
   }
 }
 
-export class WindSpeedForecastField implements ColDefGenerator, ForecastColDefGenerator {
-  private static instance: WindSpeedForecastField
-
-  static readonly field = 'windSpeed'
-  static readonly headerName = 'Wind Speed'
-  static readonly type = 'number'
-  static readonly precision = 1
-  private constructor(readonly colDefBuilder: ColumnDefBuilder) {}
-
-  public generateForecastColDef = (headerName?: string) => {
-    return this.colDefBuilder.generateForecastColDef(headerName)
-  }
-  public generateColDef = () => {
-    return this.colDefBuilder.generateColDefWith(
-      WindSpeedForecastField.field,
-      WindSpeedForecastField.headerName,
-      WindSpeedForecastField.precision
-    )
-  }
-
-  public generateColDefs = (headerName?: string) => {
-    return this.colDefBuilder.generateColDefs(headerName, true)
-  }
-
-  public static getInstance(): WindSpeedForecastField {
-    if (!WindSpeedForecastField.instance) {
-      WindSpeedForecastField.instance = new WindSpeedForecastField(
-        new ColumnDefBuilder(
-          WindSpeedForecastField.field,
-          WindSpeedForecastField.headerName,
-          WindSpeedForecastField.type,
-          WindSpeedForecastField.precision,
-          new GridComponentRenderer()
-        )
-      )
-    }
-
-    return WindSpeedForecastField.instance
-  }
-}
-
-export class PrecipForecastField implements ColDefGenerator, ForecastColDefGenerator {
-  private static instance: PrecipForecastField
-
-  static readonly field = 'precip'
-  static readonly headerName = 'Precip'
-  static readonly type = 'number'
-  static readonly precision = 1
-  private constructor(readonly colDefBuilder: ColumnDefBuilder) {}
-
-  public generateForecastColDef = (headerName?: string) => {
-    return this.colDefBuilder.generateForecastColDef(headerName)
-  }
-  public generateColDef = () => {
-    return this.colDefBuilder.generateColDefWith(
-      PrecipForecastField.field,
-      PrecipForecastField.headerName,
-      PrecipForecastField.precision
-    )
-  }
-
-  public generateColDefs = (headerName?: string) => {
-    return this.colDefBuilder.generateColDefs(headerName, false)
-  }
-
-  public static getInstance(): PrecipForecastField {
-    if (!PrecipForecastField.instance) {
-      PrecipForecastField.instance = new PrecipForecastField(
-        new ColumnDefBuilder(
-          PrecipForecastField.field,
-          PrecipForecastField.headerName,
-          PrecipForecastField.type,
-          PrecipForecastField.precision,
-          new GridComponentRenderer()
-        )
-      )
-    }
-
-    return PrecipForecastField.instance
-  }
-}
+export const TempForecastField = new IndeterminateField('temp', 'Temp', 'number', 1, true)
```

Comment:
This looks way better, thanks for the cleanup!

Just one nit:

Since the `{RH, WindDir, WindSpeed, Precip}ForecastField`s are now all variables of the `IndeterminateField` class, we can rename the exported variables with camel case but with a lowercase first letter.